pkg/config/config.go

```diff
@@ -76,57 +105,128 @@ func SaveToFile(config *Config, filename string) error {
 	return nil
 }
 
-func (c *Config) Validate() error {
+func (c *Config) Validate() []error {
+	var allTheErrors []error
+	re := regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
+	re164 := regexp.MustCompile("^\\+[1-9]\\d{1,14}$")
+
 	// validate organization
 	if c.Organization == "" {
-		return errors.New("no organization configured")
+		//return errors.New("no organization configured")
+		allTheErrors = append(allTheErrors, errors.New("no organization configured"))
 	}
 
-	re := regexp.MustCompile("^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$")
-
 	//validate users
 	userEmails := []string{}
 	for _, user := range c.Users {
 		if util.StringSliceContains(userEmails, user.PrimaryEmail) {
-			log.Fatalf("Validation failed: duplicate user defined (user: %s)", user.PrimaryEmail)
+			//return fmt.Errorf("duplicate user defined (user: %s)", user.PrimaryEmail)
```

Comment:
The commented-out return statements can be removed :)

Reply:
ooh, just leftovers, im sorry! thank you for pointing out